### ISSUE TITLE
VOLUME not updating properly, only once track is changed. 

### ISSUE THREAD

**Issue by aredwood** [spotify/web-api#572]
Issue found on February 19th 2015.

#### Endpoint(s):
GET /v1/me/player

#### Scope(s):
user-read-playback-state

#### Steps to reproduce:
1. GET endpoint with a track playing, note volume.
2. Change volume.
3. GET endpoint and note volume - it will be the same as before, despite the change.
4.  Change to another track.
5. GET endpoint will now reflect actual volume.

#### Expected behaviour:
GET endpoint should reflect proper volume on step 3, hitting end endpoint the 2nd time.

#### Actual behaviour:
Volume is only updated once track is changed (step 5).

**Comment by asmitter** [comment]
Thanks for the report. I've replicated it and we'll take a look.

**Comment by bih** [comment]
Hello @aredwood,

I can confirm this issue is fixed. I also confirmed by getting and updating the volume in both the Web API and the Desktop Client.

If you still have issues, feel free to open a new issue and reference this issue! Cheers 👍
